.changeset/hip-panthers-breathe.md

```diff
@@ -0,0 +1,5 @@
+---
+"@spectrum-css/menu": patch
+---
+
+removed keyboard focus outline in spectrum-two
```

Comment:
Can you please provide information in the PR description and changeset about why this change is being made? This is a pretty big change that affects accessibility, and needs to be communicated and validated. Peaking at the Spectrum 2 specs for menu, it does continue to have a focus indicator.